### PR TITLE
Add link to style gallery in the style manager

### PR DESCRIPTION
**Pull request by timlinux**
This change adds a link to the style gallery in the style manager.

image

### CODE REVIEW

**Comment by nyalldawson** [comment]
@nirvn can you take a look at this one?

**Comment by nirvn** [comment]
@timlinux , first: I'm super appreciative of your efforts to ease style sharing within the QGIS ecosystem, thanks. 

On the PR itself, I'm not a big fan of the current location of the label (and how small the "hit" area is with the "here" hyperlink). I see two UI alternatives:
- Add an additional item in the [ Import / export ] button menu pop up (boooouuuuuhhhh, too hidden :wink: );
- A button sitting to the right of the [ Help ] button in the standard buttons row at the bottom of the dialog.
image

I'd go for the second alternative as it'll insure maximal visibility while not getting in the way.

**Comment by m-kuhn** [comment]
Very good idea to link to these ressources from here.
I'd suggest to replace the term "here" with "style hub", the following article has some reasoning for this https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/

File: src/ui/qgsstylemanagerdialogbase.ui [hunk 1]
```diff
@@ -109,20 +113,63 @@
...
+         </property>
+         <property name="horizontalScrollBarPolicy">
+          <enum>Qt::ScrollBarAlwaysOff</enum>
+         </property>
+         <property name="html">
+          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
+&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
+p, li { white-space: pre-wrap; }
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
+&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Download more styles &lt;a href=&quot;https://plugins.qgis.org/styles/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#1b6acb;&quot;&gt;here&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (m-kuhn) [inline src/ui/qgsstylemanagerdialogbase.ui:163]:
```suggestion
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Find more styles on the &lt;a href=&quot;https://plugins.qgis.org/styles/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#1b6acb;&quot;&gt;style hub&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

**Comment by nyalldawson** [comment]
Superseded by #40974

**Pull request closed without merging**